src/server/api/controllers/orders.controller.js

```diff
@@ -0,0 +1,27 @@
+const knex = require('../../config/db');
+const HttpError = require('../lib/utils/http-error');
+
+const getOrders = async () => {
+  return knex('orders');
+};
+
+const getOrderById = async (id) => {
+  if (!id) {
```

Comment:
In this PR which is based on issue #46, you need to make a join between Users and Orders table based on user.id